sklearn/neural_network/tests/test_mlp.py

```diff
@@ -633,3 +638,213 @@ def test_n_iter_no_change_inf():
 
     # validate _update_no_improvement_count() was always triggered
     assert_equal(clf._no_improvement_count, clf.n_iter_ - 1)
+
+
+@ignore_warnings
+def check_sample_weight_invariance(name, metric, y1, y2):
```

Comment:
It would be nice if we could find a way to share rather than copy this code. Putting it in utils/testing.py might be the right thing?

Reply:
I guess so, will check if it works out.